icons/new/box checked.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`box checked.svg` seems like an unusual file name.  Are we cool with spaces in our filenames?  Would the implementation of this look like `<Icon name='box checked' />`

Reply:
All the filenames are camelCased so it’d be `boxChecked`